Week9 group5

noahblakesmith opened the pull request:
Hey team, check out my work. Feel free to edit as needed!

noahblakesmith committed:
Submit group 5 week 9 assignment

noahblakesmith committed:
Fixed table in readme.md file

noahblakesmith committed:
Attempt 2 to fix table in readme.md file

noahblakesmith committed:
Attempt 3 to fix table in readme.md file

noahblakesmith committed:
Attempt 4 to fix table in readme.md file

noahblakesmith committed:
Attempt 5 to fix table in readme.md file

MiglePetrou commented on _Week9/Group5/README.md:
```diff
@@ -0,0 +1,43 @@
+We define the true average treatment effect (ATE) as the effect of attending charters on test scores across all 5,000 children in our sample. The true ATE is simply the mean of the individual treatment effects (ITE) across all children (i.e., the difference between each child's test score attending charter and his/her test score attending public). It is mathematically equivalent to say the true A ...
+
+Why does the biased ATE differ from the true ATE? The bias is introduced by students who are admitted to charters but choose not to attend, as they would have lower test scores attending charters. As a result, the biased ATE is an overestimate of the true ATE. The latter reflects the impact of charter attendance across *all students*, whereas the biased ATE reflects the impact on the *compliers a ...
+
```
I suggest we clarify the difference between the biased ATE and the true ATE here. We state that the regression coefficient of the biased ATE represents the ATE across all students who actually attend charters, i.e., students who were admitted and then choose to attend. We also say that the true ATE reflects the impact of charter attendance across *all students*. I think this might be confusing given that our treatment is attending a charter school. Let's clarify who the "all students" are in this case: all those who were admitted, regardless of whether they choose to attend?

MiglePetrou replied:
"The bias is introduced by students who are admitted to charters but choose not to attend, as they would have lower test scores attending charters." I'd like to suggest a small edit here. There are many reasons why a student may choose not to attend a charter school in real life, but in our study we assume that potential scores are the only reason. We should state this assumption explicitly here again, or else we will inevitably receive questions about this.

f2pHgty8hw replied:
Regarding your first comment, I clarified the language some more. Note "all students" here refers to all lottery entrants, not all admits. Please update as needed if you think it needs more clarity.

You make a good point in second comment. I added that bit to the file.

MiglePetrou commented on _Week9/Group5/README.md:
```diff
@@ -0,0 +1,43 @@
+
+Why does the biased ATE differ from the true ATE? The bias is introduced by students who are admitted to charters but choose not to attend, as they would have lower test scores attending charters. As a result, the biased ATE is an overestimate of the true ATE. The latter reflects the impact of charter attendance across *all students*, whereas the biased ATE reflects the impact on the *compliers a ...
+
+Why does this discussion matter? The biased ATE reflects a model specification pioneered by Angrist, Imbens, and Rubin (1996) that has become common the charter-school literature. However, it is imperative to underline how the bias in the ATE changes the appropriate takeaways from a study using this model. Namely, a sufficiently large and significant biased ATE should only serve as evidence for p ...
```
"The biased ATE reflects a model specification pioneered by Angrist, Imbens, and Rubin (1996) that has become common **IN** the charter-school literature." -- a word is missing here.

f2pHgty8hw replied:
Updated

MiglePetrou commented on _Week9/Group5/README.md:
```diff
@@ -0,0 +1,43 @@
+
+## power-calculation.do
+We loop through our simulation 100 times. (We will do more iterations for our final project, when we have more time.) For each iteration, we add a row to a matrix with columns for true ATE, biased ATE, the upper and lower bounds of the 95% confidence interval of the biased ATE, the mean test scores for public and private, and the standard deviations in test scores for public and private. We then  ...
+
```
Let's stick to the original language, charter schools are privately run but they are still public, not private schools.

f2pHgty8hw replied:
Nice catch. Changed.

MiglePetrou commented on _Week9/Group5/README.md:
```diff
@@ -0,0 +1,43 @@
+We loop through our simulation 100 times. (We will do more iterations for our final project, when we have more time.) For each iteration, we add a row to a matrix with columns for true ATE, biased ATE, the upper and lower bounds of the 95% confidence interval of the biased ATE, the mean test scores for public and private, and the standard deviations in test scores for public and private. We then  ...
+
+Next, for each iteration (i.e., row), we calculate the minimum sample size at a 5% significance level and 0.8 power. These results are imputed into a new variable.
+
```
Do you think it would be useful to briefly explain why we are using 0.8 power?

f2pHgty8hw replied:
Added 0.8 power appears standard in the literature.

MiglePetrou commented on _Week9/Group5/README.md:
```diff
@@ -0,0 +1,43 @@
+Thereafter, we create summary statistics for the true and biased ATEs as well as the minimum sample sizes, which are exported to a .csv file.
+
+Lastly, we generate cumulative true ATEs, biased ATEs, and corresponding confidence intervals for the latter. For each row j, the cumulative variable is the mean of variable from row 1 to j. Using these new cumulative variables, we generate a graph showing the bias of the biased ATE as the sample size goes to infinity.
+
```
Nice, I love this part

MiglePetrou reviewed (changes requested):
Thank you again for offering to take the lead on this. I have reviewed all files again and left my comments. Could you please assign a group to review our work since you are the creator of this commit?

MiglePetrou commented on _Week9/Group5/estimation.do:
```diff
@@ -0,0 +1,160 @@
+replace attend = 1 if admitted==1 & ts_charter>=ts_public // =1 if child was admitted to charter and would achieve higher test score attending charter; recall hypothetical test score at charter was generated above using Cobb-Douglas education production function
+replace attend = 0 if admitted==0 | ts_charter<ts_public // =0 if child lost charter lottery or would achieve higher test score attending public
+la var attend "=1 if child attends charter"
+
```
Let's discuss if we are replacing those who reject the offer with other candidates. Do we allow any empty slots at oversubscribed schools? We probably shouldn't. Let's talk this through tomorrow.

f2pHgty8hw replied:
Updated with waitlist lottery mechanism to fill remaining seats

noahblakesmith committed:
Added waitlist lottery mechanism

Added code to estimation.do file, adjusted table and graph outputs accordingly, and adjusted readme.md file accordingly

noahblakesmith committed:
Updated graph showing bias

noahblakesmith committed:
Removed graph file

noahblakesmith committed:
Re-added graph

noahblakesmith committed:
Small edits

noahblakesmith committed:
Small edits

noahblakesmith committed:
Small edits

noahblakesmith committed:
Delete biased_and_true_ATEs_as_N_approaches_infinity.png

noahblakesmith committed:
Small changes

noahblakesmith committed:
Fixed typos

f2pHgty8hw commented:
> 

For sure. And feel free to make edits as needed. In the meantime, I will submit the review request to the other group.

noahblakesmith committed:
Updated per Migle's comments

Benjamin-Tu committed:
Update README.md

Benjamin-Tu commented on _Week9/Group5/estimation.do:
```diff
@@ -0,0 +1,194 @@
+egen waitlist = count(attend) if attend==0 & ts_charter>=ts_public, by(charter_id) // Waitlist is defined as lottery losers who still want to attend charters (i.e., higher test scores attending charters than public)
+
+*Waitlist admission (no lottery)
+replace attend = 1 if open_spots>=waitlist // When there are more spots than waitlisted students, all are granted admission
```
Added comments that explains why some seats become available.

Benjamin-Tu committed:
Update estimation.do
